lib/config/config-rule.js

```diff
@@ -299,8 +296,8 @@ function generateConfigsFromSchema(schema) {
  * @returns {rulesConfig} Hash of rule names and arrays of possible configurations
  */
 function createCoreRuleConfigs() {
-    return Object.keys(builtInRules).reduce((accumulator, id) => {
-        const rule = rules.get(id);
+    return Array.from(builtInRules.keys()).reduce((accumulator, id) => {
```

Comment:
Could this be:

```js
return Array.from(builtInRules.entries()).reduce((accumulator, [id, rule]) => {
    // ...
});
```

Then it wouldn't be necessary to define `rule` on the following line.

Reply:
Actually, I realized that this could just be 

```js
return Array.from(builtInRules).reduce((accumulator, [id, rule]) => {
    // ...
});
```

But it's not a blocker.